man/signal-cli-dbus.5.adoc

```diff
@@ -11,10 +11,11 @@ DBus API for signal-cli - A commandline and dbus interface for the Signal messen
 
 == Synopsis
 
-*signal-cli* [--config CONFIG] daemon --dbus-system
+*signal-cli* [--verbose] [--config CONFIG] [-u USERNAME] [-o {plain-text,json}] [--dbus-system] daemon
```

Comment:
for system dbus it's actually `--system` after `daemon` :sweat_smile: 
`signal-cli daemon [--system]`

Reply:
I see, thanks! Should have checked the 'main' man page :). (I've never used the system bus myself)